### PR TITLE
Fix coverage

### PR DESCRIPTION
**Pull request by abhinav**
Until goveralls is fixed

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -3,6 +3,7 @@ language: go
 
 go:
     - 1.5
+    - 1.6
```
Comment (breerly) [inline .travis.yml]: yay :)

**Comment by breerly** [comment]
wow, surprised that this is needed. :shipit: 

I guess we'll be needing this for yarpc-go as well.

**Comment by abhinav** [comment]
Oh, forgot to add the package being tested to `-coverpkg`.

File: scripts/cover.sh [hunk 2]
```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+
+set -e
+mkdir -p cover
```
Comment (prashantv) [inline scripts/cover.sh]: you probably want to delete all files in the `cover` directory if it already exists

File: scripts/cover.sh [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+PACKAGES=$(go list $(glide nv))
+QUERY='.Deps | .[] | select(startswith("github.com/thriftrw/thriftrw-go/"))'
+
+i=0
+for pkg in ${PACKAGES[@]}; do
+	i=$((i + 1))
+	dependencies=$(go list -json "$pkg" \
+		| jq -r "$QUERY" \
+		| tr '\n' ',' \
+		| sed 's/,$//')  # need to drop trailing ,
```
Comment (prashantv) [inline scripts/cover.sh]: you can do all this in jq too,
```
jq -r '.Deps | map(select(startswith("$PKG"))) | join(",")'
```

File: scripts/cover.sh (hunk 2 above)
Comment (abhinav) [inline scripts/cover.sh]: true

File: scripts/cover.sh (hunk 3 above)
Comment (abhinav) [inline scripts/cover.sh]: :+1:

**Comment by abhinav** [comment]
Updated

**Comment by prashantv** [comment]
lgtm

**Pull request merged**